### PR TITLE
chore: release v1.1.1

### PR DESCRIPTION
**Pull request by michael-siek**
## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**

- [x] Follows the commit message policy, appropriate for next version
- [x] Code is reviewed for accessibility
- [x] Code is reviewed for security

### CODE REVIEW

**Comment by jeeyyy** [comment]
@michael-siek this was simply going to be a deprecation release. Can you clarify why there are other changes?

**Review by jeeyyy — DISMISSED** [review]
Besides it could be worth adding deprecation notice to changelog too.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,22 @@
+# [DEPRECATED] axe-puppeteer
```
Comment (jeeyyy) [inline CHANGELOG.md:1]: I am not sure this is the right way to do this.

We are updating the readme with deprecation. So a changelog should simply say that this has been deprecated. Worth finding what is the best practise here. If there is none, let us not do anything

**Review by jeeyyy — DISMISSED** [review]

**Review by jeeyyy — APPROVED** [review]
## Release Audit
- [x] https://github.com/dequelabs/axe-puppeteer/pull/74
- [x] https://github.com/dequelabs/axe-puppeteer/pull/79

> Note: This release is for deprecation

**Comment by dylanb** [comment]
Reviewed

**Pull request merged**